scripts/core/editor3/components/toolbar/IconButton.tsx

```diff
@@ -20,12 +20,14 @@ interface IProps {
 export const IconButton: React.FunctionComponent<IProps> = ({onClick, iconName, tooltip, uiTheme}) => (
     <div
         data-flow={'down'}
+        data-test-id="icon-button"
+        data-test-value={tooltip}
```

Comment:
Since values are dynamic anyway here, there's no benefit to use data-test-id.

Use `getByRole('button', {name: 'abc'}`. In case of an icon button, playwright will match `name` against aria-label